lib/ColField/stories/ColFieldStory.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import Row from 'react-bootstrap/lib/Row';
+import Col from 'react-bootstrap/lib/Col';
+import ColField from '..';
+import StoryItem from '../../../stories/styleguide/StoryItem';
+import TextPreviewSVG from '../../../assets/text-preview.svg';
+
+storiesOf('Components', module).add('ColField', () => {
+  return (
+    <StoryItem title="ColField" description="stuff">
+      <Row>
+        <Col xs={12} sm={8} smOffset={2}>
+          <ColField>
+            <ColField.Header>
+              <ColField.Label editable onChange={() => {}}>
+                Howdy
+              </ColField.Label>
```

Comment:
💚 What do you think to passing `children` as a `value={}` prop instead?

I find it confusing that the children set the value, as it's not a pattern we commonly use. I think it may not be instantly obvious that this is the case when using this component.

I can see that this pattern has arisen from `<EditableTextWrapper />`

Reply:
I kept it as children since value feels a bit weird when its not 'editable', value doesn't quite make sense. Maybe instead of value, having a 'label' prop instead, that feels right for both cases then?

Reply:
I would say value is ok though, although it's not editable to the user, we are still controlling it with a value